install.sh

```diff
@@ -971,4 +731,580 @@ azk8spull() {
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests", "certificatesigningrequests/approval"]
+  verbs: ["get", "delete", "create", "update"]
+- apiGroups: ["certificates.k8s.io"]
+  resources:
+    - "signers"
+  resourceNames:
+    - "kubernetes.io/legacy-unknown"
+  verbs: ["approve"]
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["get", "create", "list", "watch", "update", "delete"]
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["mutatingwebhookconfigurations","validatingwebhookconfigurations"]
+  verbs: ["get", "create", "delete", "update", "patch"]
+- apiGroups: ["pingcap.com"]
+  resources:
+    - podchaos
+    - networkchaos
+    - iochaos
+    - timechaos
+    - kernelchaos
+    - stresschaos
+  verbs: ["*"]
+---
+# Source: chaos-mesh/templates/controller-manager-rbac.yaml
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: chaos-mesh:chaos-controller-manager
+  labels:
+    app.kubernetes.io/name: chaos-mesh
+    app.kubernetes.io/instance: chaos-mesh
+    app.kubernetes.io/component: controller-manager
+subjects:
+- kind: ServiceAccount
+  name: chaos-controller-manager
+  namespace: chaos-testing
+roleRef:
+  kind: ClusterRole
+  name: chaos-mesh:chaos-controller-manager
+  apiGroup: rbac.authorization.k8s.io
+---
+# Source: chaos-mesh/templates/controller-manager-service.yaml
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: chaos-testing
+  name: chaos-mesh-controller-manager
+  labels:
+    app.kubernetes.io/name: chaos-mesh
+    app.kubernetes.io/instance: chaos-mesh
+    app.kubernetes.io/component: controller-manager
+spec:
+  type: ClusterIP
+  ports:
+    - port: 10080
+      targetPort: http
+      protocol: TCP
+      name: http
+    - port: 443
+      targetPort: webhook
+      protocol: TCP
+      name: webhook
+  selector:
+    app.kubernetes.io/component: controller-manager
+    app.kubernetes.io/instance: chaos-mesh
+---
+# Source: chaos-mesh/templates/chaos-daemon-daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: chaos-testing
+  name: chaos-daemon
+  labels:
+    app.kubernetes.io/name: chaos-mesh
+    app.kubernetes.io/instance: chaos-mesh
+    app.kubernetes.io/component: chaos-daemon
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: chaos-mesh
+      app.kubernetes.io/instance: chaos-mesh
+      app.kubernetes.io/component: chaos-daemon
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: chaos-mesh
+        app.kubernetes.io/instance: chaos-mesh
+        app.kubernetes.io/component: chaos-daemon
+    spec:
+      hostIPC: true
+      hostPID: true
+      containers:
+        - name: chaos-daemon
+          image: pingcap/chaos-daemon:latest
+          imagePullPolicy: Always
+          command:
+            - /usr/local/bin/chaos-daemon
+            - --runtime
+            - ${runc}
+            - --http-port
+            - !!str 31766
```

Comment:
`!!str` is not needed

Reply:
If we delete `!!str`, the program will report an error. 
```
Error from server (BadRequest): error when creating "STDIN": DaemonSet in version "v1" cannot be handled as a DaemonSet: v1.DaemonSet.Spec: v1.DaemonSetSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Command: []string: ReadString: expects " or n, but found 3, error found in #10 byte of ...|tp-port",31766,"--gr|..., bigger context ...|s-daemon","--runtime","containerd","--http-port",31766,"--grpc-port",31767],"image":"pingcap/chaos-d|...
```